3p/integration.js

```diff
@@ -105,6 +106,23 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
+// used for extracting fakead3p from production code.
+const IS_DEV = true;
+
+if (IS_DEV) {
+  let toRegister = false;
+  const parseData = parseFragment(location.hash);
```

Comment:
add `getMode()` to integration.js run gulp size and get
186.76 kB   |     39.4 kB   |   13.61 kB   |   current-min/f.js / current/integration.js